kubernetes-model-generator/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/model/IntOrString.java

```diff
@@ -40,171 +34,103 @@
 
 import java.io.IOException;
 import java.io.Serializable;
-import java.util.HashMap;
-import java.util.Map;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonDeserialize(using = IntOrString.Deserializer.class)
 @JsonSerialize(using = IntOrString.Serializer.class)
-@JsonPropertyOrder({
-    "IntVal",
-    "Kind",
-    "StrVal"
-})
 @ToString
 @EqualsAndHashCode
 @Setter
 @Accessors(prefix = {
     "_",
     ""
 })
-@Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage=true, builderPackage = "io.fabric8.kubernetes.api.builder")
 public class IntOrString implements Serializable {
 
-    @JsonProperty("IntVal")
-    private Integer IntVal;
-    @JsonProperty("Kind")
-    private Integer Kind;
-    @JsonProperty("StrVal")
-    private String StrVal;
-    @JsonIgnore
-    private Map<String, Object> additionalProperties = new HashMap<>();
-
-    public IntOrString() {
-    }
-
-    //Builders are generated for the first non-empty constructor found.
-    public IntOrString(Integer intVal, Integer kind, String strVal, Map<String, Object> additionalProperties) {
-        IntVal = intVal;
-        Kind = kind;
-        StrVal = strVal;
-        this.additionalProperties = additionalProperties;
-    }
+  private transient Object value;
```

Comment:
Why is this transient? If serialized using Java Serialization it will be ignored (which renders the class useless).